vocabularies/gsq-alias.ttl

```diff
@@ -0,0 +1,117 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/gsq-alias> a owl:Ontology .
+
+als:conceptScheme a skos:ConceptScheme ;
+    dct:created "2020-03-23T16:19:50+10:00"^^xsd:dateTime ;
+    dct:creator <http://linked.data.gov.au/org/gsq> ;
+    dct:modified "2020-03-23T16:38:10+10:00"^^xsd:dateTime ;
+    dct:source "Compiled by the Geological Survey of Queensland" ;
+    skos:definition "Sources for any alternate names for features, sites, surveys, or samples catalogued by the Geological Survey of Queensland."@en ;
+    skos:hasTopConcept als:colloquial,
+        als:des,
+        als:geoscience-australia,
+        als:gsq-legacy,
+        als:igsn,
+        als:mmol,
+        als:ogia,
+        als:operator,
+        als:service-provider,
+        als:treasury ;
+    skos:prefLabel "GSQ Alias"@en .
+
+als:colloquial a skos:Concept ;
+    skos:altLabel "Common"@en,
+        "Informal"@en,
+        "Popular"@en,
+        "Public"@en ;
+    skos:definition "An unofficial, informal, or colloquial name or reference used by the general public, industry, or government."@en ;
+    skos:inScheme als:conceptScheme ;
+    skos:prefLabel "Colloquial"@en ;
+    skos:topConceptOf als:conceptScheme .
+
+als:gsq-legacy a skos:Concept ;
+    skos:definition "Any superseded identifier, code, or name previously used by the Geological Survey of Queensland."@en ;
+    skos:inScheme als:conceptScheme ;
+    skos:prefLabel "GSQ Legacy"@en ;
+    skos:topConceptOf als:conceptScheme .
+
+als:igsn a skos:Concept ;
+    skos:altLabel "International Geo Sample Number"@en ;
+    skos:definition "An identifier used as a globally unique and persistent identifier for material samples."@en ;
+    skos:inScheme als:conceptScheme ;
+    skos:prefLabel "IGSN"@en ;
+    skos:topConceptOf als:conceptScheme .
+
+als:des a skos:Concept ;
+    skos:altLabel "Department of Environment and Science"@en ;
+    skos:definition "An identifier used by the Department of Environment and Science"@en ;
+    skos:inScheme als:conceptScheme ;
+    skos:prefLabel "DES"@en ;
+    skos:topConceptOf als:conceptScheme .
+
+als:geoscience-australia a skos:Concept ;
+    skos:altLabel "G.A."@en ;
+    skos:definition "An identifier used by Geoscience Australia"@en ;
+    skos:inScheme als:conceptScheme ;
+    skos:prefLabel "Geoscience Australia"@en ;
+    skos:topConceptOf als:conceptScheme .
+
+als:mmol a skos:Concept ;
+    skos:altLabel "MyMinesOnline"@en ;
+    skos:definition "An identifier used by the MyMinesOnline system administered by the Department."@en ;
+    skos:inScheme als:conceptScheme ;
+    skos:prefLabel "MMOL"@en ;
+    skos:topConceptOf als:conceptScheme .
+
+als:ogia a skos:Concept ;
+    skos:altLabel "Office of Groundwater Impact Assessment"@en ;
+    skos:definition "An identifier used by the Office of Groundwater Impact Assessment."@en ;
+    skos:inScheme als:conceptScheme ;
+    skos:notation "QWRC" ;
+    skos:prefLabel "OGIA"@en ;

```

Comment:
I think that the prefLabel is QWRC - that will allow us to continue to show the QWRC number in GeoResGlobe.

Reply:
done. QWRC pref label. Full name and OGIA as altLabels